ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/T8nSubCommand.java

```diff
@@ -0,0 +1,557 @@
+/*
+ * Copyright ConsenSys AG.
```

Comment:
Old copyright

Reply:
If ConsenSys were to migrate these to the new format we could have spotless verify and then magically apply them to new files.

Reply:
Fair point